src/modules/core/components/avatar/avatar.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { Avatar } from './avatar';
+
+const meta: Meta<typeof Avatar> = {
+  title: 'Core UI/Avatar',
+  component: Avatar,
+  tags: ['autodocs'],
+};
+
+export default meta;
+type Story = StoryObj<typeof Avatar>;
+
+export const Default: Story = {
+  args: {
+    // url: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSPeg-dzIDOCGO3HF0p__2il1jVuAQ5XfTEjw&usqp=CAU',
```

Comment:
You can create two stories here, one Default which will use all default data, without event size and shape. And another one will use all custom data.